src/wallet/wallet.cpp

```diff
@@ -6936,3 +7337,137 @@ bool CMerkleTx::AcceptToMemoryPool(const CAmount &nAbsurdFee, CValidationState &
 
 bool CompSigmaHeight(const CSigmaEntry &a, const CSigmaEntry &b) { return a.nHeight < b.nHeight; }
 bool CompSigmaID(const CSigmaEntry &a, const CSigmaEntry &b) { return a.id < b.id; }
+
+namespace {
+//I am leaving this proc in place as it may be useful for further urgent cases.
```

Comment:
What was the original urgent case? Comment doesn't look good. Please consider removing the function altogether.

Reply:
It is Lelantus disabled.

Reply:
I'll save it into some other repo